build_tools/bazel_to_cmake/bazel_to_cmake.py

```diff
@@ -86,15 +95,40 @@ def setup_environment():
       os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
 
+def repo_relpath(path):
+  return os.path.relpath(path, repo_root)
+
+
 def log(*args, **kwargs):
   print(*args, **kwargs, file=sys.stderr)
```

Comment:
You could make some of the indented logs below slightly less ugly if you allow something like this:

```python
      log(
          f"\nERROR.\n"
          f"Missing a rule handler in bazel_to_cmake_converter.py?\n"
          f"Reason: `{type(e).__name__}: {e}`",
          indent=2)  # or prefix="  ", but we probably want it to always be spaces.
```

Reply:
Ooh that's a nice idea. I was kind of hoping that one day we would find a better logging library than my custom wrapper around pring. But every time I look into python logging it looks awful.